corehq/apps/zapier/tests.py

```diff
@@ -138,7 +141,31 @@ def test_subscribe(self):
         self.assertIsNotNone(subscription.repeater_id)
         self.assertNotEqual(subscription.repeater_id, '')
 
-    def test_unsubscribe(self):
+    def test_subscribe_case(self):
+
+        data = {
+            "subscription_url": ZAPIER_URL,
+            "target_url": ZAPIER_URL,
+            "event": consts.EventTypes.NEW_CASE,
+            "case": CASE_TYPE
+        }
+        response = self.client.post(reverse('zapier_subscribe', kwargs={'domain': self.domain}),
```

Comment:
You can replace the `"zapier_subscribe"` string literal with `SubscribeView.urlname`, which will make it easier to change that name in the future if we ever decide to do so.